diffkemp/semdiff/custom_pattern_config.py

```diff
@@ -71,7 +79,53 @@ def get_extension(path):
         _, ext = os.path.splitext(path)
         return ext.lower()[1:]
 
-    def add_pattern(self, path):
+    @staticmethod
+    def _get_clang_cpattern_options():
+        """
+        Returns the options specific to compiling C patterns.
+        """
+        diffkemp_patterns_path = os.path.abspath(
+            os.path.join(os.path.dirname(__file__), "../simpll"))
```

Comment:
I think that we should place the header outside of the SimpLL directory as it had nothing to do with SimpLL. Maybe create a new `patterns` directory?

Reply:
```CPatternPass.cpp``` and ```CustomPatternSet.cpp```, which are part of the SimpLL library, include this header, that's why I placed it there. However, I can still create the new ```patterns``` directory if you think it's preferable.

Reply:
Thinking about it more, I think that we could consider the header as an API of DiffKemp (the first one we have). Therefore, I believe that it belongs to the `include/` directory, maybe I'd go as far as put it into `include/diffkemp/patterns.h` and then we'd include it with `#include <diffkemp/patterns.h>` (sounds a bit nicer than `#include <diffkemp_patterns.h>`, at least to me).